docker-examples/minimum-poetry/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM python:3.11-slim
+
+ARG POETRY_VERSION=1.8
+
+# See https://python-poetry.org/docs/#ci-recommendations
+RUN pip install "poetry==${POETRY_VERSION}"
+
+WORKDIR /app
+
+# --- Reproduce the environment ---
+# You can comment the following two lines if you prefer to manually install
+#   the dependencies from inside the container.
+COPY pyproject.toml .
+
+RUN poetry install
```

```suggestion
RUN poetry install --no-root
```
Otherwise it might fail.

---

It actually worked for me without it, but I added it anyway as  you suggested, as it doesn't go against the main function. Thanks